hmcts/cnp-flux-config
# Neuvector sbox testing

To test neuvector with autoscaler on sbox
**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```
